spring-integration-sftp/src/main/java/org/springframework/integration/sftp/outbound/SftpMessageHandler.java

```diff
@@ -63,7 +63,7 @@ public SftpMessageHandler(SftpRemoteFileTemplate remoteFileTemplate, FileExistsM
 	 * (SessionFactory)
 	 */
 	public SftpMessageHandler(SessionFactory<LsEntry> sessionFactory) {
-		super(sessionFactory);
+		this(new SftpRemoteFileTemplate(sessionFactory));
```

Comment:
Well, the `SftpMessageHandler` has been introduced since 4.3, although I think it won't hurt to back port it into 4.2 and 4.1.
I'll prepare the backport commit when the review will be ready already. Without docs, of course.